### ISSUE TITLE
Provide minified version

### ISSUE THREAD

**Issue by niutech** [wintermute-cell/magick.css#12]
Please provide magick.min.css via unpkg.com and fix your CDN instructions:
```
<link rel="stylesheet" href="https://unpkg.com/normalize.min.css">
<link rel="stylesheet" href="https://unpkg.com/magick.min.css">
```

**Comment by PatriceRoot** [comment]
I need to work on this issue but I need more clarifications

**Comment by wintermute-cell** [comment]
Thanks @PatriceRoot but there is nothing you can do here, I'll have to address this myself as soon as I find the time :)

**Comment by PatriceRoot** [comment]
Okay @wintermute-cell. Good luck 😉

**Comment by wintermute-cell** [comment]
fixed in 65ef870ec49c391f108704a65190ff4f1650979c and 381d364175a456479a41c72c7bd483921534e811
